sdk/search/Azure.Search/src/Generated/Models/AutocompleteRequest.cs

```diff
@@ -10,7 +12,7 @@ public partial class AutocompleteRequest
     {
         /// <summary> The search text on which to base autocomplete results. </summary>
         public string SearchText { get; set; }
-        /// <summary> Specifies the mode for Autocomplete. The default is &apos;oneTerm&apos;. Use &apos;twoTerms&apos; to get shingles and &apos;oneTermWithContext&apos; to use the current context in producing autocomplete terms. </summary>
+        /// <summary> Specifies the mode for Autocomplete. The default is &apos;oneTerm&apos;. Use &apos;twoTerms&apos; to get shingles and &apos;oneTermWithContext&apos; to use the current context while producing auto-completed terms. </summary>
```

Comment:
You've probably noticed, but Search updated their swagger so changes like this are expected.